### PR TITLE
check that docker image exists already, use latest as 1.1.2 tag no lo…

### PR DESCRIPTION
**Pull request by factorypreset**
…nger exists

### CODE REVIEW

File: tasks/snapshot [hunk 1]
```diff
@@ -25,6 +25,11 @@ if [ ! $(command -v docker) ]; then
   exit 1
 fi
 
+if [[ "$(docker images -q brew/dynamodump 2> /dev/null)" == "" ]]; then
```
Comment (MattSurabian) [inline tasks/snapshot]: Should this be `bchew`?

Comment (factorypreset) [reply tasks/snapshot]: yes it should

File: tasks/restore [hunk 2]
```diff
@@ -42,6 +42,11 @@ if [ ! $(command -v docker) ]; then
   exit 1
 fi
 
+if [[ "$(docker images -q brew/dynamodump 2> /dev/null)" == "" ]]; then
```
Comment (MattSurabian) [inline tasks/restore]: also should be `bchew`

Comment (factorypreset) [reply tasks/restore]: copypasta fail. fixed..

**Comment by MattSurabian** [comment]
Come to think of it, now that we have the version off the docker run command, can't we skip checking for the image at all since Docker run will automatically download it if it's not there?

**Pull request merged**